spec/features/idv/end_to_end_idv_spec.rb

```diff
@@ -2,16 +2,21 @@
 
 RSpec.describe 'Identity verification', :js do
   include IdvStepHelper
+  include InPersonHelper
 
   # Needed specs:
   # unsupervised proofing happy path mobile only
   # hybrid mobile end to end (edit hybrid_mobile_spec)
   # verify by mail
   # in person proofing
```

Comment:
We can take these two comments out now

Reply:
Good point. I removed the whole comment and added LG-10495 for the hybrid_mobile spec. Should we add a ticket for a verify by mail spec that doesn't use in_person, or do you think that's covered well enough?

Reply:
I feel verify_by_mail is covered well with IPP. The only difference would be the account is active right before the personal key page.